nnpdf_data/nnpdf_data/utils.py

```diff
@@ -1,12 +1,9 @@
 import pathlib
 
-import ruamel.yaml as yaml
+from ruamel.yaml import YAML
 from validobj import ValidationError, parse_input
 
-try:
-    Loader = yaml.CLoader
-except AttributeError:
-    Loader = yaml.Loader
+yaml = YAML(typ='safe')
```

Comment:
Using libyaml (CLoader) when available makes a huge difference in time when loading large (or many) files.

Reply:
CLoader is PyYaml, do we want to keep using both yaml packages?

Reply:
I could not find an explicit mention in the docs but it might be that as long as you use `safe` (and purre is not True) it will use CLoader under the hood https://sourceforge.net/p/ruamel-yaml/tickets/159/#8f94 

but the comment is from 2017... we should check. The difference in performance was enough to think about dropping ruamel if we can no longer use CLoader